docs/kubernetes/kctlr-f5-resource.rst

```diff
@@ -0,0 +1,104 @@
+   data                    A JSON object                                            Required
+   ----------------------- -------------------------------------------------------- -----------
+   - frontend              Defines BIG-IP objects.
+   ----------------------- -------------------------------------------------------- -----------
+   - backend               Identifies the Service you want to proxy.
+
+                           Defines a BIG-IP health monitor(s) for the Service.
+   ----------------------- -------------------------------------------------------- -----------
+   f5type                  A ``label`` property watched by the |kctlr|.             Optional
+   ----------------------- -------------------------------------------------------- -----------
+   schema                  Tells the |kctlr| how to interpret the encoded data.     Required
+   ======================= ======================================================== ===========
+
+.. _f5 resource frontend:
+
+Data.Frontend
+`````````````
+
+The :code:`data.frontend` property defines the objects you want to create on the BIG-IP system for a specific Service. You can use either the :code:`virtualServer` properties or the :code:`iApp` properties.
+
+You can define :code:`frontend` using the `k8s-bigip-ctlr virtual server configuration parameters`_ or the `k8s-bigip-ctlr iApp configuration parameters`_.
+
+The iApp configuration parameters include a set of customizable ``iappVariables`` parameters. These custom user-defined parameters must correspond to fields in the iApp template you want to launch. You can also define the `iApp pool member table`_ that the iApp creates on the BIG-IP system.
+
+.. tip:: With the iApp configuration parameters, you can also deploy objects across BIG-IP modules (for example, ASM and LTM).
+
+.. _f5 resource backend:
+
+Data.Backend
+````````````
+
+Use the :code:`data.backend` property to identify the `Kubernetes Service`_ that makes up the back end server pool.
+
+.. tip:: You can also define BIG-IP health monitors in this section.
+
+
+F5 type
+```````
+
+.. important::
+
+   The usage of the :code:`f5type` property differs depending on whether you're using it in Kubernetes or OpenShift.
+
+In an F5 Resource ConfigMap, the :code:`f5type` value must be :code:`virtual-server`. This tells the |kctlr| that you want it to create a BIG-IP virtual server.
+
+In an OpenShift Route resource, the :code:`f5type` property functions as a :code:`route-label`. You can use it as an identifier for Routes that you want the |kctlr| to watch. This is useful for multi-Controller deployments managing traffic for multiple Applications (and/or Projects) within a single BIG-IP partition.
+
+.. _f5-schema:
+
+F5 schema
+`````````
+
+The `F5 schema`_ allows the |kctlr| to communicate with the BIG-IP system. While all |kctlr| versions are backwards-compatible, using an older schema may limit Controller functionality.
```

Comment:
The schema isn't necessarily involved in communication but provides validation of structure and data provided in the ConfigMap.

This might be a better explanation: "The `F5 schema`_ defines the structure and formatting for the data provided by the F5 resource. The |kctlr| will use the provided version to validate the inputs."